File: erlang/euler1.erl
-module(euler1).
-export[euler1/0].

% Sum of all number below 1000 divisible by 3 or 5
euler1() ->
    lists:foldl(
        fun(X, Sum) -> X + Sum end, 
        0, 
	[X || X <- lists:seq(1, 999), X rem 3 == 0 orelse X rem 5 == 0]).

File: java/euler-1/src/main/java/no/gjetanger/sandbox/Euler1.java
package no.gjetanger.sandbox;

public final class Euler1 {

    public static void main(String[] args ) {
        int sum = 0;
        for (int i=1; i<1000; i++) {
            if (i % 3 == 0 || i % 5 == 0) {
                sum += i;
            }
        }
        System.out.println(String.format("Sum of all integers divisible by 3 or 5 below 1000: %d", sum));
    }

}
